### DISCUSSION TITLE
@xterm/addon-web-links doesn't support blob URL (URI)

### DISCUSSION THREAD

**Discussion by terryzfeng** [sourcelair/xterm.js#5046] [Q&A] [answered]
Using the addon for weblinks, my blob links get highlighted as blob:[https://url.com](https://url.com) instead of [blob:https://url.com](https://url.com)

## Details
- Browser and browser version: 
  - Chrome 
- xterm.js version: 
  - "@xterm/addon-web-links": "^0.11.0",
  - "xterm": "^5.3.0",

### Steps to reproduce

1. Create a blob URL and print it to the terminal

**Comment by jerch** [comment, marked as answer]
The addon handles only HTTP(S) on purpose by default. If thats not what you want, you can customize the regexp to your needs, see https://github.com/xtermjs/xterm.js/blob/master/addons/addon-web-links/src/WebLinksAddon.ts

**Reply by terryzfeng** [reply]
Thanks I was able to register a link provider customizing this! Appreciate it!
